content/api/power-board.md

```diff
@@ -19,4 +19,28 @@ r.power_board.power_off()
 `power_on` is called when you setup your robot, so this doesn't need to be called manually. The ports will come on automatically as soon as your robot is ready, before the start button is pressed.
 {{% /notice %}}
 
+## Buzzing :bee:
+
+The power board has a piezo buzzer which can beep.
+
+The `buzz` function accepts multiple parameters, depending on what you want to play. The first argument is the duration of the beep, in seconds. The later arguments are either the note you want to play, or the buzzers frequency.
```

Comment:
Worth mentioning that the frequency is in Hz? (Rather than being angular frequency which would be another reasonable choice).

Reply:
Maybe something about the frequency range too? It's obvious to us that the piezo can only sensibly buzz in audible frequencies but that isn't necessarily obvious to teams.

Reply:
I would really hope that's obvious, but i'll add something just in case